src/memgraph.cpp

```diff
@@ -520,8 +520,9 @@ int main(int argc, char **argv) {
   std::optional<CoordinatorState> coordinator_state{std::nullopt};
   auto const is_valid_data_instance =
       coordination_setup.management_port && !coordination_setup.coordinator_port && !coordination_setup.coordinator_id;
-  auto const is_valid_coordinator_instance =
-      coordination_setup.management_port && coordination_setup.coordinator_port && coordination_setup.coordinator_id;
+  auto const is_valid_coordinator_instance = coordination_setup.management_port &&
+                                             coordination_setup.coordinator_port && coordination_setup.coordinator_id &&
+                                             !coordination_setup.coordinator_hostname.empty();
```

Comment:
you have an exception message a few lines after so that should be adjusted as well